add CORB post for monday (#3025)
oktadeveloper/okta.github.io: _source/_posts/2019-08-26-combat-side-channel-attacks-with-corb.md

---
layout: blog_post
title: "Combat Side-Channel Attacks with Cross-Origin Read Blocking"
author: tom-abbott

> description: "Find out what those CORB errors mean and how CORB protects your application from hardware-layer vulnerabilities"

Reviewer: Add period at end of description.

Even if you don't include a `nosniff` command, CORB will still do its best to weed out inappropriate files. It does this by sniffing the response to work out its content type. This is a backup, though, to help with incorrectly labeled cross-origin responses, based on a best-effort heuristic approach. Using `nosniff` should still be your primary means of protection.

## Use CORB to Address Side-Channel Vulnerabilities

CORB is a thoughtful response to Spectre and Meltdown, which changed the security requirements for web developers. Before these persistent side-channel vulnerabilities came to light, most web developers only needed to focus on application-level security. Afterward, they had to consider how the browser managed its memory in the request lifecycle—that represented a new and daunting task for many developers.

CORB enables the browser to do the heavy lifting for the developer. Okta is glad to support it and protect its customers and partners.

## Read More About Security

For more information about browser development and security, check out these other posts from the Okta Developer blog:

> * [Bootiful Development with Spring Boot and React  ](/blog/2017/12/06/bootiful-development-with-spring-boot-and-react)

Reviewer: ```suggestion
* [Bootiful Development with Spring Boot and React](/blog/2017/12/06/bootiful-development-with-spring-boot-and-react)
```

> * [10 Excellent Ways to Secure Your Spring Boot Application ](/blog/2018/07/30/10-ways-to-secure-spring-boot)

Reviewer: ```suggestion
* [10 Excellent Ways to Secure Your Spring Boot Application](/blog/2018/07/30/10-ways-to-secure-spring-boot)
```

> * [Build a Web App with Spring Boot and Spring Security in 15 Minutes ](/blog/2018/09/26/build-a-spring-boot-webapp)

Reviewer: ```suggestion
* [Build a Web App with Spring Boot and Spring Security in 15 Minutes](/blog/2018/09/26/build-a-spring-boot-webapp)
```

> Leave a comment below, find more from the Okta team on [Twitter](https://twitter.com/oktadev), and watch us on [YouTube](https://www.youtube.com/channel/UC5AMiWqFVFxF1q9Ya1FuZ_Q)!

Reviewer: ```suggestion
Leave a comment below, find more from the Okta team on [Twitter](https://twitter.com/oktadev), and watch us on [YouTube](https://www.youtube.com/c/oktadev)!
```
